src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useCallback } from 'react';
+import debounce from 'lodash.debounce';
```

Comment:
Fix it
<img width="889" alt="image" src="https://github.com/mate-academy/react_autocomplete/assets/64750958/ed2bce7f-0e93-4f96-9dc6-97e86d3df5c6">